test/suites/build/build.go

```diff
@@ -1,22 +1,99 @@
 package build
 
 import (
+	"fmt"
 	"path/filepath"
+	"strconv"
+	"strings"
 
 	"github.com/alibaba/sealer/test/testhelper"
+	"github.com/alibaba/sealer/test/testhelper/settings"
+	"github.com/alibaba/sealer/utils"
+	"github.com/onsi/gomega"
 )
 
-func getFixtures() string {
-	pwd := testhelper.GetPwd()
-	return filepath.Join(pwd, "suites", "build", "fixtures")
+func GetFixtures() string {
+	return filepath.Join(testhelper.GetPwd(), "suites", "build", "fixtures")
 }
 
-func GetOnlyCopyDir() string {
-	fixtures := getFixtures()
-	return filepath.Join(fixtures, "only_copy")
+func GetLocalBuildDir() string {
+	return "local_build"
 }
 
-func GetBuildTestDir() string {
-	fixtures := getFixtures()
-	return filepath.Join(fixtures, "build_test")
+func GetCloudBuildDir() string {
+	return "cloud_build"
+}
+
+func GetTestKubeKile() string {
+	// only copy
+	return ""
+}
+
+func GetTestContext() string {
+	return ""
+}
+func GetTestBuildType() string {
+	return ""
+}
+
+func GetTestImageName() string {
+	return fmt.Sprintf("sealer-io/%s%d:%s", settings.ImageName, 719, "v1")
+}
+
+func GetTestClusterFilePath() string {
+	return ""
+}
+
+type ArgsOfBuild struct {
+	KubeFile, ImageName, Context, BuildType string
+}
+
+func (a *ArgsOfBuild) SetKubeFile(kubeFile string) *ArgsOfBuild {
+	a.KubeFile = kubeFile
+	return a
+}
+
+func (a *ArgsOfBuild) SetImageName(imageName string) *ArgsOfBuild {
+	a.ImageName = imageName
+	return a
+}
+
+func (a *ArgsOfBuild) SetContext(context string) *ArgsOfBuild {
+	a.Context = context
+	return a
+}
+
+func (a *ArgsOfBuild) SetBuildType(buildType string) *ArgsOfBuild {
+	a.BuildType = buildType
+	return a
+}
+
+func (a *ArgsOfBuild) ToString() string {
+	return fmt.Sprintf("%s build -f %s -t %s -c %s -b %s", settings.DefaultSealerBin, a.KubeFile, a.ImageName, a.Context, a.BuildType)
+}
+
+func NewArgsOfBuild() *ArgsOfBuild {
+	return &ArgsOfBuild{}
```

Comment:
NewArgsOfBuild is too simple, I think let call do "&ArgsOfBuild{}" would be fine.